fe/fe-core/src/main/java/org/apache/doris/datasource/CatalogMgr.java

```diff
@@ -462,6 +464,20 @@ public CatalogIf replayCreateCatalog(CatalogLog log, boolean isReplay) throws Dd
         writeLock();
         try {
             CatalogIf catalog = CatalogFactory.constructorFromLog(log);
+            Map<String, String> props = log.getProps();
+            if (props.containsKey(METADATA_REFRESH_INTERVAL_SEC)) {
```

Comment:
This should be done after `((ExternalCatalog) catalog).checkProperties()`.
And all properties, including `METADATA_REFRESH_INTERVAL_SEC`, should be checked in `checkProperties()`.
After `checkProperties()`, all properties should be valid.